### DISCUSSION TITLE
Expat version in 2.88.6 and 2.88.7 releases?

### DISCUSSION THREAD

**Discussion by sjmathews** [mono/SkiaSharp#2820] [Q&A] [answered]
I'm trying to determine the version of expat in our application to triage vulnerabilities. Could you please tell me what version of expat the 2.88.6 and 2.88.7 releases are using or possibly how to find this information? Thank you!

**Comment by mattleibow** [comment, marked as answer]
It is 2.5.0: https://chromium.googlesource.com/external/github.com/libexpat/libexpat.git/+/441f98d02deafd9b090aea568282b28f66a50e36/expat/
